src/scene/skybox/sky.js

```diff
@@ -59,6 +75,19 @@ class Sky {
         this.projectedSkydomeCenterId = this.device.scope.resolve('projectedSkydomeCenter');
     }
 
+
+    applySettings(render) {
+        if (render.skyPosition) {
+            this.position = new Vec3(render.skyPosition[0], render.skyPosition[1], render.skyPosition[2]);
+        }
+        if (render.skyScale) {
+            this.scale = new Vec3(render.skyScale[0], render.skyScale[1], render.skyScale[2]);
+        }
+        if (render.skyTripod) {
```

Comment:
I think lets use name `skyCenter` instead of `skyTripod` to match the naming inside the engine